pkg/infra/container/imagecontext/base/Dockerfile

```diff
@@ -19,7 +19,8 @@ COPY 10-network-security.conf /etc/sysctl.d/
 ARG PASSWORD="Seadent123"
 
 RUN echo "Installing Packages ..." \
-    && sed -i "s/archive.ubuntu.com/mirrors.aliyun.com/g" /etc/apt/sources.list \
+    # && sed -i "s/archive.ubuntu.com/mirrors.aliyun.com/g" /etc/apt/sources.list \
+    && sed -i -e 's/archive.ubuntu.com\|security.ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list \
```

Comment:
It looks like `old-releases.ubuntu.com` is an older one just from the name. Is this the best choice?

Reply:
Sealer can run well on base-image using ubuntu:21.04 but other versions still have some problem and I don't know why yet. So I choice ubuntu:21.04, and the life cycle of Ubuntu:21.04 releases ends in January 2022, so there will be an error when executing `apt-get`.Use `old-releases.ubuntu.com` can fix this problem.